netlogo-gui/src/main/app/App.scala

```diff
@@ -847,6 +832,13 @@ class App extends
     showAboutWindow()
   }
 
+  /**
+   * This is called reflectively by the mac app wrapper.
+   */
+  def handleShowPreferences(): Unit = {
+    showPreferencesDialog()
```

Comment:
I basically followed the way handleShowAbout is done, but why is this indirection good? Why not have mac-app call `showPreferencesDialog` and `showAboutMenu`?